pyproject.toml

```diff
@@ -36,6 +36,7 @@ python-multipart = "^0.0.6"
 webrtcvad = "^2.0.10"
 pyaudio = "^0.2.14"
 halo = "^0.0.31"
+alembic = "^1.13.1"
```

Comment:
Can you make sure to add the equivalent line to requirements-windows.txt?
It would be, I think: alembic~=1.13.1
And should immediately follow the last line before the PyTorch dependencies (i.e., right after the halo line). I tested it out and it seems to be importable at least.

Reply:
I can't test windows can you add it. 

'And should immediately follow the last line before the PyTorch dependencies (i.e., right after the halo line). I tested it out and it seems to be importable at least.' what does this mean?

Reply:
I added it if you can test